lua/ide/workspaces/workspace_controller.lua

```diff
@@ -246,25 +246,41 @@ function WorkspaceController.new(config)
         -- issues, so do it only after "VimEnter" fires.
         local restore = libwin.restore_cur_win()
         local init_autocmd = nil
-        init_autocmd =  vim.api.nvim_create_autocmd({ "VimEnter" }, {
-            callback = function()
-                -- do not assign if we are git tool.
-                local buf_name = vim.api.nvim_buf_get_name(0)
-                if vim.fn.match(buf_name, ".git/*") >= 0 then
+        local function init_callback()
+            -- do not assign if we are git tool.
+            if string.find(vim.api.nvim_buf_get_name(0), "%.git/") then
```

Comment:
What i mean is leave the original lines with vim.fn.match

Reply:
local buf_name = vim.api.nvim_buf_get_name(0)
                 if vim.fn.match(buf_name, ".git/*") >= 0 then
         local function init_callback()
         
Do notneed to change

Reply:
oh gotcha, fixed